src/SignalR/clients/java/signalr/test/src/main/java/com/microsoft/signalr/GsonHubProtocolTest.java

```diff
@@ -56,6 +56,15 @@ public void parsePingMessage() {
         assertEquals(HubMessageType.PING, messages.get(0).getMessageType());
     }
 
+    @Test
+    public void writeCloseMessage() {
+        CloseMessage closeMessage = new CloseMessage();
+        String result = TestUtils.byteBufferToString(hubProtocol.writeMessage(closeMessage));
+        String expectedResult = "{\"type\":7,\"allowReconnect\":false}\u001E";
```

Comment:
Why does this message have allowReconnect: false

Reply:
GsonHubProtocol just calls Json.write(object), we haven't put in any custom logic for serializing yet. And I don't really want to right now just to avoid this extra field